src/dappsExplorer/search/DAppsExplorerScreenSearch.test.tsx

```diff
@@ -347,8 +353,7 @@ describe(DAppsExplorerScreenSearch, () => {
       expect(within(allDappsSection).getByText(dappsList[0].name)).toBeTruthy()
     })
 
-    it('triggers events when searching', async () => {
-      jest.useRealTimers()
+    it('triggers events when searching', () => {
```

Comment:
Strangely, I couldn't make it work leaving `async`.
It's not needed here so I removed, but still could be confusing in the future if we add `async` again.